packages/react/src/components/Storage/StorageManager/utils/uploadFile.ts

```diff
@@ -8,7 +8,7 @@ interface UploadFileProps {
   isResumable?: boolean;
   progressCallback: (progress: { loaded: number; total: number }) => void;
   errorCallback: (error: string) => void;
-  completeCallback: (event: { key: string }) => void;
+  completeCallback: (event: { key?: string }) => void;
```

Comment:
This is the correct type actually:
https://github.com/aws-amplify/amplify-js/blob/main/packages/storage/src/types/AWSS3Provider.ts#L82

Reply:
Does that mean `event` is just an empty object when there is no value for `key`? Seems like if there was no value ` key` then `event` itself would just be `undefined` be empty but 🤷  

For improved TS practices, what do you think of typing as:
```suggestionx
  completeCallback: (event: { key: string | undefined }) => void;
```

Reply:
Makes sense to me, will update